docs/source/status.rst

```diff
@@ -94,7 +94,7 @@ Data Types
 +===================+=======+=======+=======+============+=======+=======+=======+
 | Dictionary        | ✓     | ✓ (1) | ✓     | ✓ (1)      | ✓ (1) | ✓ (1) | ✓     |
 +-------------------+-------+-------+-------+------------+-------+-------+-------+
-| Extension         | ✓     | ✓     | ✓     |            |       |       | ✓     |
+| Extension         | ✓     | ✓     | ✓     |            |       | ✓     | ✓     |
```

Comment:
Not sure about this one, but we support propagating the metadata which seems to be all that is required??